imageio/plugins/ffmpeg.py

```diff
@@ -534,13 +614,41 @@ def _initialize(self):
                    '-pix_fmt', self._pix_fmt,
                    '-r', "%.02f" % fps,
                    '-i', '-', '-an',
-                   '-vcodec', codec] 
-            cmd += ['-b', str(bitrate)] if (bitrate is not None) else [] 
-            cmd += ['-r', "%d" % fps, self._filename]
-            
+                   '-vcodec', codec,
+                   '-pix_fmt', pixelformat,
+                   ]
+            if bitrate is not None:
+                cmd += ['-b:v', str(bitrate)]
+            if quality >= 0:  # If < 0, then we don't add anything
```

Comment:
I put the -1 option as a way to remove all variable bit rate and fixed bit rate params to ffmpeg. Some codecs require different parameters for compression so you don't want those. This way if someone uses a different codec that needs a different compression parameter they can manually specify it with ffmpeg_params without getting a conflict. 

But maybe there is a better way to do this that would be more straight forward? Any suggestions?